src/mbgl/annotation/annotation_tile.cpp

```diff
@@ -19,37 +19,105 @@ AnnotationTile::~AnnotationTile() {
     annotationManager.removeTile(*this);
 }
 
-void AnnotationTile::setNecessity(Necessity) {}
+void AnnotationTile::setNecessity(Necessity) {
+}
+
+class AnnotationTileFeatureData {
+public:
+    AnnotationTileFeatureData(const AnnotationID id_,
+                              FeatureType type_,
+                              GeometryCollection geometries_,
```

Comment:
Is the reason this is not `GeometryCollection && geometries_,` is that `clang-tidy` claims that copying is faster here than explicitly asking for a move?